sljit_src/sljitProtExecAllocator.c

```diff
@@ -83,6 +83,7 @@ struct chunk_header {
        as it only uses local variables
 */
 
+#include <sys/stat.h>
```

Comment:
Is this available in all posix OSes?

Reply:
yes, eventhough I was only considering Linux and NetBSD for this codebase
https://pubs.opengroup.org/onlinepubs/009695399/basedefs/sys/stat.h.html